app/ui/src/app/integrations/edit-page/action-configure/action-configure.component.ts

```diff
@@ -223,6 +223,16 @@ export class IntegrationsConfigureActionComponent extends FlowPage
       this.continue({});
       return;
     }
+    const anyVisibleProperty = Object.values(this.formConfig).reduce((a, c) => a && 'hidden' !== c.type, true);
```

Comment:
I think it would be good to replace the single letter variable names with something a little more descriptive. This allows for better readability and makes bugs easier to find when they do occur.